apps/fz_http/lib/fz_http_web/templates/shared/user_details.html.heex

```diff
@@ -45,9 +45,11 @@
       <td><%= FzHttp.Devices.count(@user.id) %></td>
     </tr>
 
-    <tr>
-      <td><strong>Number of Rules</strong></td>
-      <td><a href={"#{@rules_path}"}><%= FzHttp.Rules.count(@user.id) %></a></td>
-    </tr>
+    <%= if @rules_path do %>
```

Comment:
I'm conditionally showing this here, I figured just admin users would want to see the number of rules.

Reply:
👍